tests/.suitespec.json

```diff
@@ -3,7 +3,9 @@
         "$setup": [
             "setup.py",
             "setup.cfg",
-            "pyproject.toml"
+            "pyproject.toml",
+            "hatch.toml"
```

Comment:
currently this file doesn't seem to contain build/setup configuration? `$harness` is probably more appropriate if we want to run the full suite whenever this source is modified